OpenSim/Actuators/Millard2012AccelerationMuscle.cpp

```diff
@@ -809,7 +811,7 @@ void Millard2012AccelerationMuscle::
                         muscleName.c_str());
 
                 setActuation(s, 0.0);
-                setFiberLength(s,m_penMdl.getOptimalFiberLength());
+                setFiberLength(s, get_optimal_fiber_length());
```

Comment:
Writing to `cerr` above and below are almost pointless. Useful for diagnosing a bad simulation afterwards, but what if the user doesn't realize the simulation is garbage and thing the muscle is producing zero tension and its optimal fiber-length?